src/shared/services/UnreadCounterService.ts

```diff
@@ -75,6 +87,12 @@ export class UnreadCounterService {
         );
       }
     }
+  }
+
+  public updateAll = async () => {
```

Comment:
This one also.

Reply:
This still needs to be a closure so it can be properly used in `poll(this.updateAll, updateUnreadCountsInterval);`.

Reply:
You could bind it in the constructor.

Reply:
Ah okay. That's fine then.